src/EnergyPlus/Autosizing/Base.cc

```diff
@@ -623,7 +623,7 @@ void BaseSizer::overrideSizingString(std::string &string)
     this->overrideSizeString = false;
 }
 
-Real64 BaseSizer::setOAFracForZoneEqSizing(EnergyPlusData &state, Real64 const &desMassFlow, DataSizing::ZoneEqSizingData const &zoneEqSizing)
+Real64 BaseSizer::setOAFracForZoneEqSizing(EnergyPlusData &state, Real64 const desMassFlow, DataSizing::ZoneEqSizingData const &zoneEqSizing)
```

Comment:
In this file, I'm not sure whether we want the `Real64` to pass by const ref to avoid floating precision errors. (the int/bool for sure we don't)